go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/oschwald/maxminddb-golang v1.10.0
 	github.com/pion/dtls/v2 v2.2.4
 	github.com/pkg/errors v0.9.1
-	github.com/quic-go/quic-go v0.32.0
+	github.com/quic-go/quic-go v0.33.0
```

Comment:
Since it requires Go 1.19 now, can you bump the version at the top of the file?

Reply:
Will do